workflow/rules/damage.smk

```diff
@@ -6,6 +6,7 @@ rule MapDamage:
         bam="results/BOWTIE2/{sample}/AlignedToPathogenome.bam",
     params:
         pathogenome_path=config["pathogenome_path"],
```

Comment:
Change to `pathogenome_path=os.path.dirname(config["pathogenomesFound"])`

Reply:
Yes, agreed!